apps/lockfile-explorer-web/src/parsing/getPackageFiles.ts

```diff
@@ -3,14 +3,16 @@
 
 import { IPackageJson } from '../types/IPackageJson';
 
+const serviceUrl: string = window.appContext.serviceUrl;
+
 /**
  * Fetches a projects configuration files from the local file system
  *
  * @returns a json object representing a package.json or a text file to be rendered (in the case of readPnpmfile)
  */
 export const readPnpmfile = async (): Promise<string> => {
   try {
-    const response = await fetch(`http://localhost:8091/api/pnpmfile`);
+    const response = await fetch(`${serviceUrl}/api/pnpmfile`);
```

Comment:
Shouldn't this always be the same as the server hosting the page?

Reply:
@iclanton No, for development Webpack serves the bundle on a different port from the Node.js service.